free-1.md

```diff
@@ -45,6 +59,8 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+A similar service, [Foodpairing](https://www.foodpairing.com/en/home) already exists. However, it requires users to sign up and pay. I signed up for the free trial and it allows users to receive pairings based on foods they input. However, it also tries to let users create their own pairings. All of this is done through a confusing graph interface, where each node in the graph is a different type of food. I think it has a good database of foods and flavor pairings but the interface could definitely be improved.
```

Comment:
I like how you actually tried the service! The required sign up itself immediately turned me away from even trying. Perhaps it's not so bad that users can create their own pairings. Maybe there could be a pre-existing database of well-established flavor pairings and the option to allow users to vote on good pairings that aren't necessarily common knowledge. I'm also not entirely sure how subjective flavor pairing can be, as I'm sure some people have unique quirks.